tests/DoctrineSnapshotAdapterTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+/*
+ * This file is part of the prooph/service-bus.
+ * (c) 2014 - 2015 prooph software GmbH <contact@prooph.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Date: 10/10/15 - 15:37
+ */
+
+namespace ProophTest\EventStore\Adpater\MongoDb;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Schema\Schema;
+use PHPUnit_Framework_TestCase as TestCase;
+use Prooph\EventStore\Aggregate\AggregateType;
+use Prooph\EventStore\Snapshot\Adapter\Doctrine\DoctrineSnapshotAdapter;
+use Prooph\EventStore\Snapshot\Snapshot;
+
+/**
+ * Class MongoDbAdapterTest
+ * @package ProophTest\EventStore\Adpater\Doctrine
+ */
+final class DoctrineSnapshotAdapterTest extends TestCase
+{
+    /**
+     * @var DoctrineSnapshotAdapter
+     */
+    private $adapter;
+
+    /**
+     * @var Connection
+     */
+    private $connection;
+
+    protected function setUp()
+    {
+        $connection = [
+            'driver' => 'pdo_sqlite',
+            'dbname' => ':memory:'
+        ];
+
+        $this->connection = DriverManager::getConnection($connection);
+
+        $schema = new Schema();
+        $table = $schema->createTable('foo_snapshot');
+        $table->addColumn('aggregate_type', 'string');
+        $table->addColumn('aggregate_id', 'string');
+        $table->addColumn('last_version', 'integer');
```

Comment:
SnapshotSchema helper would be nice like we have for the event store doctrine adpater: https://github.com/prooph/event-store-doctrine-adapter/blob/master/src/Schema/EventStoreSchema.php